src/pages/products/Products.js

```diff
@@ -1,10 +1,81 @@
+import { Link } from 'react-router-dom';
+import { getProducts, updateProduct } from 'services';
 import './products.scss';
 
 export class Products extends Component {
+  state = {
+    products: [],
+  }
+
+  componentDidMount() {
+    getProducts()
+      .then(products => this.setState({ products }));
+  }
+
+  deleteProduct = (e, id) => {
+    e.preventDefault();
+  };
+
+  onChange = ({ target }, id) => {
+    this.setState((prevState) => {
+      const updatedProducts = [...prevState.products];
+      updatedProducts.map(item => item.title = item.id === id ? target.value : item.title);
```

Comment:
1) why use `map` here?
2) you need to change only one item, for what reason iterate all items?